docs/src/main/asciidoc/keycloak-guide.adoc

```diff
@@ -334,8 +334,51 @@ export access_token=$(\
 )
 ```
 
+== Multi-Tenant Deployments
+
+The Keycloak Quarkus Extension also supports multitenancy. This capability is specially useful if your service is serving different
```

Comment:
we should be consistent about how `multitenancy` is spelled.

Reply:
Good point. I change all references to `multitenancy`.